spaceship/lib/spaceship/tunes/app_ratings.rb

```diff
@@ -0,0 +1,107 @@
+module Spaceship
+  module Tunes
+    # Represents app ratings from iTunesConnect
+    class AppRatings < TunesBase
+      # @return (Spaceship::Tunes::Application) A reference to the application
+      #   this version is for
+      attr_accessor :application
+
+      # @return (Spaceship::Tunes::AppRatingSummary) A summary of the overall ratings for the application
+      attr_accessor :rating_summary
+
+      # @return (Hash) mapping country codes to a (Spaceship::Tunes::AppRatingSummary) summary of ratings for that country
+      attr_reader :store_fronts
+
+      # @return (Hash) of iTunesConnect version id's to readable version numbers
+      attr_reader :versions
+
+      attr_mapping({
+        'versions' => :versions
+      })
+
+      class << self
+        # Create a new object based on a hash.
+        # This is used to create a new object based on the server response.
+        def factory(attrs)
+          obj = self.new(attrs)
+
+          obj.unfold_rating_summary(attrs['ratings'])
+          obj.unfold_store_fronts(attrs['storeFronts'])
+
+          return obj
+        end
+      end
+
+      def unfold_rating_summary(attrs)
+        unfolded_rating_summary = AppRatingSummary.new(attrs)
+        instance_variable_set(:@rating_summary, unfolded_rating_summary)
+      end
+
+      def unfold_store_fronts(attrs)
+        unfolded_store_fronts = {}
+
+        attrs.each do |info|
+          unfolded_store_fronts[info['countryCode']] = AppRatingSummary.new(info['ratings'])
+        end
+
+        instance_variable_set(:@store_fronts, unfolded_store_fronts)
```

Comment:
I wasn't sure about the best way of going about this, I copied the approach from somewhere else in spaceship. Could you give me a little more info on what would be a better way of doing it? Like I said I'm not super familiar with ruby.  Thanks!